### PR TITLE
Average word length vignette added

### PR DESCRIPTION
**Pull request by kphaterp**

### CODE REVIEW

File: vignettes/avg_word_len.Rmd [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+You can start by loading the `textfeatureinfo` package:
+```{r setup}
+library(textfeatureinfor)
+```
+
+## Quick example for one piece of text
+
+
+To understand the basics of the `avg_word_len` function, we can take a look at it's interaction with a simple text object.
```
Comment (Jacq4nn) [inline vignettes/avg_word_len.Rmd]:
```suggestion
To understand the basics of the `avg_word_len` function, we can take a look at its interaction with a simple text object. 
```

File: vignettes/avg_word_len.Rmd [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+toy_data["average_word_lengths"] <- round(toy_data["average_word_lengths"], 2)
+toy_data
+```
+
+Now we have access to the average length of the words for each example message in the toy data frame.
+
+## Summary
+
+
+The `avg_word_len` function is simple, but has broach implications. It allows for easier extraction of useful text information in one line of code. This feature is useful for natural language processing and may prove useful when building machine learning models.
```
Comment (Jacq4nn) [inline vignettes/avg_word_len.Rmd]:
```suggestion
The `avg_word_len` function is simple, but has broad implications. It allows for easier extraction of useful text information in one line of code. This feature is useful for natural language processing and may prove useful when building machine learning models.
```

**Review by Jacq4nn — CHANGES_REQUESTED** [review]
Hi Kiran! Great work. Just added two comments as there were typos

File: vignettes/avg_word_len.Rmd [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+The `avg_word_len` function from the `textfeatureinfo` package is one of four functions that aids in this text data extraction process. Specifically, `avg_word_len` makes it easy to extract the average length of words inside of a piece of text. 
+
+This document provides detailed instructions on how to apply the `avg_word_len` function on text data for quick access to a new feature that can be added into various machine learning models.
+
+You can start by loading the `textfeatureinfo` package:
+```{r setup}
+library(textfeatureinfor)
+```
+
+## Quick example for one piece of text
```
Comment (Jacq4nn) [inline vignettes/avg_word_len.Rmd]:
```suggestion
## Quick example with one piece of text
```

**Review by lynnwbl — APPROVED** [review]

**Review by Jacq4nn — APPROVED** [review]

**Pull request merged**